### ISSUE TITLE
"Our Ethos" section is difficult to understand

### ISSUE THREAD

**Issue by sjakobi** [haskellfoundation/haskell.foundation#50]
I'm a non-native English speaker who uses English everyday, but I had some difficulty understanding the "our ethos" section on the front page:

> Haskell’s design puts principle ahead of expediency by cleaving closely to the principles of purely functional programming.

I can't remember whether I've ever come across the word "expediency" before. Nor am I familiar with the expression "to cleave closely to sth.".

> HF sees itself as a means to augment, celebrate, and coordinate the contributions and leadership of Haskell’s vibrant community.

I feel that "augment" as a verb is also a rather obscure word. Maybe use "boost" or "leverage"?!

Lastly, the word "ethos" itself is rather unclear. Maybe the section could be titled "our values"?!

**Comment by goldfirere** [comment]
Good point -- and my guess is that there are a number of native English speakers who would find similar issues but may not be confident enough to speak up. I propose

> Haskell's design puts principle ahead of what is easy or popular by being firmly guided by the principles of purely functional programming.

Not as [mellifluous](https://www.merriam-webster.com/dictionary/mellifluous), perhaps, but more accessible.

And

> HF sees itself as a means to aid, celebrate, and coordinate the contributions and leadership of Haskell's vibrant community.

I'm also fine with "our values" over "our ethos". 

@simonpj may have an opinion.

**Comment by simonpj** [comment]
Thanks for raising this.  All these words, however wonderful, are ineffective they are not understood.

I'm happy with "Our values" and with "Haskell's design puts principle ahead of what is easy or popular by being firmly guided by the principles of purely functional programming."

 > HF sees itself as a means to augment, celebrate, and coordinate the contributions and leadership of Haskell’s vibrant community.

Maybe "amplify"?   "Boost" is ok too.   "Leverage" sounds a bit self-serving.  Or, more explicitly but maybe more clumsy

* HF sees itself as a means to celebrate, coordinate, and make more powerful and effective, the contributions and leadership of Haskell’s vibrant community.

We want to get across that HF's role is to make everyone more effective in what they do, not suppliant them.

**Comment by sjakobi** [comment]
Thanks for being so receptive about this :)

> Haskell's design puts principle ahead of what is easy or popular by being firmly guided by the principles of purely functional programming.

A tad long, I think, but much better than the current phrase!

> HF sees itself as a means to aid, celebrate, and coordinate the contributions and leadership of Haskell's vibrant community.

I like this! @simonpj's version is a bit too long and grammatically complex IMHO. "Boost" instead of "aid" would also sound good IMHO.
